packages/components/src/Form/src/Form.tsx

```diff
@@ -0,0 +1,102 @@
+import {
+    ButtonContext,
+    TextFieldContext
+} from "@hopper-ui/components";
+import {
+    useResponsiveValue,
+    useStyledSystem,
+    type StyledComponentProps,
+    type ResponsiveProp
+} from "@hopper-ui/styled-system";
+import clsx from "clsx";
+import { forwardRef, type ForwardedRef, type CSSProperties } from "react";
+import { useContextProps, Form as RACForm, type FormProps as RACFormProps } from "react-aria-components";
+
+import { cssModule, SlotProvider } from "../../utils/index.ts";
+
+import { FormContext } from "./FormContext.ts";
+
+import styles from "./Form.module.css";
+
+export const GlobalFormCssSelector = "hop-Form";
+
+export interface FormProps extends StyledComponentProps<RACFormProps> {
+    /**
+     * Whether the form elements are disabled.
+     */
+    isDisabled?: boolean;
+
+    /**
+     * If `true`, the TextField will take all available width.
```

Comment:
```suggestion
     * If `true`, the Form will take all available width.
```
Unless you mean all the fields will be the ones that are full width?